tests/users/profile.test.js

```diff
@@ -0,0 +1,168 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import app from "../../index";
+
+chai.use(chaiHttp);
+
+describe("Test /profiles", () => {
+  let token;
+  before("Login and return a token", done => {
```

Comment:
Where are you creating this user with email ```luc.bayo@gmail.co``

Reply:
Also, make sure to clean the db, after testing.

Reply:
<img width="602" alt="screen shot 2019-02-13 at 17 57 00" src="https://user-images.githubusercontent.com/20681465/52724844-c8270880-2fb8-11e9-8556-c5d03c3549d9.png">